parl/env/atari_wrappers.py

```diff
@@ -282,7 +295,11 @@ def step(self, action):
 
     def reset(self, **kwargs):
         obs = self._env.reset(**kwargs)
-        if self._get_curr_episode() == self._end_episode:
+        # remove stick equality to prevent infinite loop
+        # this only affects the mock example tests because during noop reset, it is possible that more than 3 done signals are emitted by the 
+        # mock env which uses done = random.choice(True, False, p=[0.1, 0.9]). However, it is impossible for the true env to emit more than 3
+        # done signals in one noop reset
```

Comment:
During the noop reset in NoopResetEnv, env may be reset multiple times(may occur in mock env, almost impossible in atari env), so the `==` condition may never be met. `>=` can avoid infinite loop.